code42v2.json

```diff
@@ -1546,16 +1904,14 @@
             "action": "create user",
             "identifier": "create_user",
             "description": "Create a new Code42 user account",
-            "verbose": "There is a short delay before you will be able to add the new user to departing employee or high risk employee list.",
+            "verbose": "There is a short delay before you will be able to add the new user to departing employee or high risk employee list. For the password parameter: Defaults to no password set, which leaves user in 'invited' state. They will need to reset password via email prior to first login. <b>Note:</b> If the provided username already exists for a user, it will be updated in the database instead.",
```

Comment:
```suggestion
            "verbose": "There is a short delay before you will be able to add the new user to the departing employee or high risk employee lists. For the password parameter: Defaults to no password set, which leaves user in 'invited' state. They will need to reset password via email prior to first login. <b>Note:</b> If the provided username already exists for a user, it will be updated in the database instead.",
```

Reply:
Fixed